ui/details-panel/AnnotationsTab.jsx

```diff
@@ -1,4 +1,5 @@
 import PropTypes from 'prop-types';
+import { react2angular } from "react2angular/index";
```

Comment:
In this case `"react2angular/index"` is the same as `"react2angular", since the package's `package.json` defines `main` as `index.js`:
https://github.com/coatue-oss/react2angular/blob/d0b8c336b2a52e05839636b7651dad959be76e7a/package.json#L5
...which makes the `/index` redundant.

However looking at the package there is also a newer ES2015 (aka ES6) version of the file, which we can use instead:
https://unpkg.com/react2angular@3.2.1/index.es2015.js

(compare to https://unpkg.com/react2angular@3.2.1/index.js which has been generated for older browsers)

As such, I think we should use (have fixed the quotes too):
`import { react2angular } from 'react2angular/index.es2015';`

Reply:
Very cool!  Thanks for finding that.  Fixed.